app/src/main/java/com/globant/example/mentorapp/home/presentation/view/adapter/ListUsersAdapter.java

```diff
@@ -0,0 +1,66 @@
+package com.globant.example.mentorapp.home.presentation.view.adapter;
+
+import java.util.List;
+
+import com.globant.example.mentorapp.R;
+import com.globant.example.mentorapp.home.presentation.model.ModelUserEntity;
+import com.squareup.picasso.Picasso;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+/**
+ * Custom adapter to show General data fro users.
+ * Created by oscar.amezquita on 8/06/2017.
+ */
+
+public class ListUsersAdapter extends RecyclerView.Adapter<ListUsersAdapter.ViewHolder> {
+
+    private List<ModelUserEntity> users;
+    private Context context;
+
+    public ListUsersAdapter(List<ModelUserEntity> users, Context context) {
+        this.users = users;
+        this.context = context;
+    }
+
+    @Override
+    public ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.list_users_adapter, null);
+        return new ViewHolder(view);
+    }
+
+    @Override
+    public void onBindViewHolder(ViewHolder holder, int position) {
+        ModelUserEntity user = users.get(position);
+        holder.userNameTextView.setText(user.getName());
+        if (!user.getImageUrl().isEmpty()) {
```

Comment:
I put this validation because I don't know if all users profile has an image. But never happens. Validation erased.